### PR TITLE
ENG-107 Add karate tests into build

### PR DESCRIPTION
**Pull request by jknight-liatrio**

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -1,8 +1,8 @@
 apply plugin: 'java'
 apply plugin: 'maven'
 
-group = 'accelerate'
-version = '0.1.3'
+group = 'com.liatrio.accelerate'
+version = '0.0.1-SNAPSHOT'
 
 description = """accelerate-aqe"""
```
Comment (stein321) [inline build.gradle]: only need normal quotes here

File: pom.xml [hunk 2]
```diff
@@ -23,13 +32,27 @@
 			<version>1.2.5</version>
 		</dependency>
 
-		<!-- These are the junit dependencies -->
+        <!-- These are the junit dependencies -->
 		<dependency>
```
Comment (stein321) [inline pom.xml]: let's get rid of the above comments and keep the pom file clean. I'm guessing junit should have the test scope, but I could be wrong here.

File: src/test/java/examples/ExamplesTest.java [hunk 3]
```diff
@@ -0,0 +1,11 @@
+package examples;
```
Comment (stein321) [inline src/test/java/examples/ExamplesTest.java]: generally, the package is the same as the groupID-artifact id for consistency, but not required

File: pom.xml (hunk 2 above)
Comment (jknight-liatrio) [reply pom.xml]: Good point. I updated the comments to be more meaningful.

**Pull request merged**